components/server/src/main/java/com/hotels/styx/server/netty/connectors/HttpPipelineHandler.java

```diff
@@ -706,27 +711,11 @@ protected void channelRead0(ChannelHandlerContext ctx, HttpRequest request) thro
         }
     }
 
-    private static final class HttpInterceptorContext implements HttpInterceptor.Context {
-        private final Map<String, Object> context = new ConcurrentHashMap<>();
-        private final boolean secure;
-
-        public HttpInterceptorContext(boolean secure) {
-            this.secure = secure;
+    private static InetSocketAddress remoteAddress(ChannelHandlerContext ctx) {
+        if (ctx.channel() instanceof EmbeddedChannel) {
```

Comment:
Might be worth adding a comment that this is just for unit testing?

Reply:
I only moved this method, it's not new.